### PR TITLE
Romanian translation for diverse files part 1

### PR DESCRIPTION
**Pull request by AndreiMiloiu**

### CODE REVIEW

File: dll/cpl/desk/lang/ro-RO.rc
Comment (reactos-contributor) [inline dll/cpl/desk/lang/ro-RO.rc:16]: The term I used when translating „save” was „păstrează”, because in English it includes meanings like „save money”, or „save (your) breath”, whereas in Romanian it doesn't. Also, take note that, becausi it is just an intermediary action, you've already agreed to use long infinitives, so may result as „Păstrează &ca…”.

File: dll/shellext/mydocs/lang/ro-RO.rc
Comment (reactos-contributor) [inline dll/shellext/mydocs/lang/ro-RO.rc:5]: I translated „folder” as „dosar”, and I strongly recommend you to not leave it untranslated (like Microsoft did, unfortunately).

**Review by reactos-contributor — CHANGES_REQUESTED** [review]

File: dll/cpl/desk/lang/ro-RO.rc
Comment (reactos-contributor) [inline dll/cpl/desk/lang/ro-RO.rc:17]: I seemingly forgot about this. Please avoid assigning access keys to letters with diacritics (and also for „s” and „t”, that can have diacritics under them), see 17th point in [Romanian translation notes.txt](https://github.com/reactos/reactos/blob/master/media/doc/Romanian%20translation%20notes.txt)

**Review by reactos-contributor — CHANGES_REQUESTED** [review]

**Comment by AndreiMiloiu** [comment]
I fixed the issues. Sorry for my late reply.

File: dll/cpl/desk/lang/ro-RO.rc
Comment (reactos-contributor) [inline dll/cpl/desk/lang/ro-RO.rc:132]: There is a problem with access key assignation to letter „p” here.
First, it's because for this form it was already assigned on the control with label „&Pictograme mari”, therefore for it to properly work you should assign other letter. Just for the record, in this form the access keys assigned so far are „a”, „c”, „e”, „f”, „m”, „r”, „p”, and „u”. One of the unassigned letters is, for example, „o”.
Second, regarding assigning access keys in general, Microsoft advises us to „avoid using letters that are difficult to see when they are assigned to access keys” then enlists „letters with descenders, such as p, g, q, or y” (see [Guidelines for Keyboard User Interface Design](https://learn.microsoft.com/ro-ro/previous-versions/windows/desktop/dnacc/guidelines-for-keyboard-user-interface-design?redirectedfrom=MSDN#atg_keyboardshortcuts_selecting_access_keys), which has also been included at the 10th point in [Romanian translation notes.txt](https://github.com/reactos/reactos/blob/master/media/doc/Romanian%20translation%20notes.txt)).

File: dll/shellext/sendmail/lang/ro-RO.rc
Comment (reactos-contributor) [inline dll/shellext/sendmail/lang/ro-RO.rc:5]: Elsewhere in the codebase, „Desktop” is translated as „Birou”. Either translate it here too, or consider updating the rest of the translations.

**Review by reactos-contributor — CHANGES_REQUESTED** [review]
I understand that this is still considered to be work in progress? Otherwise it's hard to understand the first control with a big text left untranslated in the IDD_THEMES form.

File: dll/cpl/desk/lang/ro-RO.rc
Comment (JoachimHenze) [inline dll/cpl/desk/lang/ro-RO.rc:2]: When adding your name here, then please also add an email address for others to contact you. Otherwise it is rather useless to add your name here. Furthermore I do recommend to add yourself at the bottom, since you did not create the file.

File: dll/cpl/powercfg/lang/ro-RO.rc
Comment (JoachimHenze) [inline dll/cpl/powercfg/lang/ro-RO.rc:2]: Same: Add your email and move yourself to the bottom.

**Review by JoachimHenze — CHANGES_REQUESTED** [review]

File: dll/cpl/powercfg/lang/ro-RO.rc
Comment (binarymaster) [reply dll/cpl/powercfg/lang/ro-RO.rc:2]: I think I will just update these to our new standard headers before merging.

Comment (reactos-contributor) [reply dll/cpl/powercfg/lang/ro-RO.rc:2]: The same intervention may be applicable to [PR №4699](https://github.com/reactos/reactos/pull/4699) and [PR №4700](https://github.com/reactos/reactos/pull/4700) as well.

**Comment by AndreiMiloiu** [comment]
I fixed all the issued. Yes, the work is still in progress. I didn't have time to translate more. I totally forgot about that long text line. Thank you everyone for your help!

File: dll/shellext/mydocs/lang/ro-RO.rc
Comment (binarymaster) [inline dll/shellext/mydocs/lang/ro-RO.rc:13]: `ro-RO.rc` file is created, but not added to the place where other .rc files are present.

File: dll/shellext/sendmail/lang/ro-RO.rc
Comment (binarymaster) [inline dll/shellext/sendmail/lang/ro-RO.rc:14]: Same for this one.

Comment (AndreiMiloiu) [reply dll/shellext/sendmail/lang/ro-RO.rc:14]: So, what should I do with these two files? Shouldn't they be placed in lang folder?

Comment (AndreiMiloiu) [reply dll/shellext/sendmail/lang/ro-RO.rc:14]: Or the problem is the header?

Comment (binarymaster) [reply dll/shellext/sendmail/lang/ro-RO.rc:14]: The same as I explained earlier: https://github.com/reactos/reactos/pull/4693#issuecomment-1247355524

**Review by JoachimHenze — COMMENTED** [review]
My concerns have been addressed.

**Review by JoachimHenze — APPROVED** [review]

File: dll/shellext/sendmail/lang/ro-RO.rc
Comment (AndreiMiloiu) [reply dll/shellext/sendmail/lang/ro-RO.rc:14]: Ok, I will edit the files.

**Comment by AndreiMiloiu** [comment]
Ok, I did it.

**Comment by AndreiMiloiu** [comment]
How should I translate the word "controller" in Romanian language?

**Comment by reactos-contributor** [comment]
_„How should I translate the word «controller» in Romanian language?”_

It depends on the context. In one of the files touched by you in this pull request, the one for [joystick GUI translation](https://github.com/reactos/reactos/blob/master/dll/cpl/joy/lang/ro-RO.rc), „controller” is in fact short for „controller device”, which in Romanian has been translated in its extended form (as you can see - „dispozitiv de control”). I recommend to either continue translate it likewise (when found in the same context), or (in case you decide to translate it differently) to take care of all the places where it has the same meaning.
I said that it depends on the context. Another case, which I presume you may encounter and remain uncertain about, is „domain controller” (see [net_msg.mc](https://github.com/reactos/reactos/blob/master/sdk/include/reactos/mc/net_msg.mc), [netmsgmsg.mc](https://github.com/reactos/reactos/blob/master/sdk/include/reactos/mc/netmsgmsg.mc), and [errcodes.mc](https://github.com/reactos/reactos/blob/master/sdk/include/reactos/mc/errcodes.mc)). In this case „controller”, as per [Wikipedia's explanation](https://en.wikipedia.org/wiki/Domain_controller), refers to a „server computer”. So, in order to avoid the risk of confusion, „domain controller” may be translated again in a more extended form, even if it may be a bit verbose (e.g. „server pentru controlul domeniilor”, „server de control domenii”, or something close).

**Comment by AndreiMiloiu** [comment]
Ok, thank you very much!

File: base/applications/mstsc/lang/ro-RO.rc
Comment (GeoB99) [inline base/applications/mstsc/lang/ro-RO.rc:70]: To be frank I'm not keen on literal translations of words between languages. Whilst `desk` means `birou`, that's for sure, `desktop` in question actually refers to `PC`, `calculator` or just simply desktop as the average Romanian speaker with basic of understanding of tech knows what does this stuff mean. `Birou` gives weird vibes to be honest.

Either translate it as `Desktop de la distanță` or just `Calculator de la distanță`. I would opt more for the former suggestion. I do not think we are supposed to translate everything to Romanian just for the sake of it.

**Review by GeoB99 — COMMENTED** [review]
I haven't looked at MC translations yet, the rest seems OK to me.

File: base/applications/mstsc/lang/ro-RO.rc
Comment (reactos-contributor) [reply base/applications/mstsc/lang/ro-RO.rc:70]: I also agree for this particular suggestion - „calculator la distanță” sounds better adapted (besides translated). Now, there is a bit of work to touch the rest of the Romanian resources where it still is translated like that.
Just for the record, the simple „Desktop” I still find better translated as „Birou”.

**Comment by reactos-contributor** [comment]
I looked over the work and I found places for which I have change suggestions. I'll come back later today with a review.

File: dll/cpl/powercfg/lang/ro-RO.rc
Comment (AndreiMiloiu) [reply dll/cpl/powercfg/lang/ro-RO.rc]: I corrected the mistake.

**Comment by AndreiMiloiu** [comment]
I hope I have corrected every mistake. Please review and retest.

File: dll/win32/syssetup/lang/ro-RO.rc
Comment (reactos-contributor) [inline dll/win32/syssetup/lang/ro-RO.rc:125]: Please consider avoiding both „a salva” and „setări” (see 15th point in [Romanian translation notes.txt](https://github.com/reactos/reactos/blob/master/media/doc/Romanian%20translation%20notes.txt)) For translating „settings” maybe „opțiuni” also serves as a good alternative ehm... option.

File: sdk/include/reactos/mc/pciclass.mc
Comment (reactos-contributor) [inline sdk/include/reactos/mc/pciclass.mc:38]: „Dispozitiv de control SCSI” (i.e. drop „al domeniilor” when it's about a low level computing part or protocol; and given the fact that this entire file „pci class” refers to the [PCI (hardware) data bus](https://en.wikipedia.org/wiki/Peripheral_Component_Interconnect) class of pluggable hardware boards, it means dropping it for everything in here).

**Review by reactos-contributor — CHANGES_REQUESTED** [review]

**Comment by AndreiMiloiu** [comment]
I fixed everything. I made more mistakes because of my misattention. Thank you for the help! I think everything looks good now.

File: dll/cpl/sysdm/lang/ro-RO.rc
Comment (reactos-contributor) [inline dll/cpl/sysdm/lang/ro-RO.rc:318]: The access key for „e” is still assigned twice (down from three).

**Review by reactos-contributor — APPROVED** [review]
I think is good enough now.

File: dll/cpl/sysdm/lang/ro-RO.rc
Comment (AndreiMiloiu) [reply dll/cpl/sysdm/lang/ro-RO.rc]: It's fixed now.

**Review by binarymaster — COMMENTED** [review]
Few things for maintainers (and for me) before merging:
- [x] Verify that there is no whitespace at line endings
- [ ] Verify copyright years for authors which were added by checking git history
- [ ] Verify dialog/windows element positions to make sure there are no overflows

File: dll/shellext/zipfldr/lang/ro-RO.rc
Comment (binarymaster) [inline dll/shellext/zipfldr/lang/ro-RO.rc:57]: **Critical:** Two buttons are placed in one position (probably by accident, will fix that).

File: dll/win32/msgina/lang/ro-RO.rc
Comment (binarymaster) [inline dll/win32/msgina/lang/ro-RO.rc:210]: These are graphical (owner-drawn) buttons, no need for reposition.

File: dll/win32/msgina/lang/ro-RO.rc
Comment (binarymaster) [inline dll/win32/msgina/lang/ro-RO.rc:213]: Left untranslated (will fix).

**Review by binarymaster — APPROVED** [review]

**Pull request merged**
